### DISCUSSION TITLE
ios18.0.1 When can  receive support

### DISCUSSION THREAD

**Discussion by wangdahua576635956** [SideStore/SideStore#710] [Q&A]
found that it can't be opened when I refresh the sidestore, software version 0.57-0.58 and new bate, ios18.0.1
﻿
ios18.0.1 when I refresh the sidestore，sidestore no opened！！！急急急
﻿
Instructions to reproduce
In iOS 18.0.1, the problem can be reproduced by refresh the sidestore
﻿
What version of SideStore are you using?
0.57,0.58,new bate

**Comment by Demonyger** [comment]
> found that it can't be opened when I refresh the sidestore, software version 0.57-0.58 and new bate, ios18.0.1
> 
> ﻿
> 
> ios18.0.1 when I refresh the sidestore，sidestore no opened！！！急急急
> 
> ﻿
> 
> Instructions to reproduce
> 
> In iOS 18.0.1, the problem can be reproduced by refresh the sidestore
> 
> ﻿
> 
> What version of SideStore are you using?
> 
> 0.57,0.58,new bate
